jobserver/runtime.py

```diff
@@ -12,3 +12,9 @@ class Runtime:
     minutes: int = attr.ib(default=0, validator=[less_than_60])
     seconds: int = attr.ib(default=0, validator=[less_than_60])
     total_seconds: int = attr.ib(default=0)
+
+    def __bool__(self):
+        if self.hours == 0 and self.minutes == 0 and self.seconds == 0:
+            return False
+
+        return True
```

Comment:
Yeah, this is what always gets me with `any()` (and `all()`), I always have to think carefully that they do what I want.  I think I'll leave this as is because it's not saving _that_ much to flip it and as we've both found it's just a little less readable.